database/schema.sql

```diff
@@ -17,7 +17,7 @@ CREATE TABLE IF NOT EXISTS users (
   firstname varchar NOT NULL,
   lastname varchar NOT NULL,
   email varchar NOT NULL,
-  profilepic_URL varchar,
+  profilepic_URL varchar
```

Comment:
I didn't check but I"m assuming you're using PostgreSQL.

Why are you using varchar instead of text? Using varchar w/o a max is non-standard SQL just like text, however, by using text you're demonstrating that you know what you're doing vs this which could just be that you don't know how it's working in postgres